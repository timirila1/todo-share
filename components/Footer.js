import React from 'react';
import { StyleSheet, View } from 'react-native';
import { colors, dims } from '../services/consts';

export const Footer = props => (
    <View style={styles.footer}>
        {props.children}
    </View>
);

const styles = StyleSheet.create({
    footer: {
        borderTopWidth: 1,
        borderTopColor: colors.border,
        position: 'absolute',
        height: dims.footerHeight,
        flex: 1,
        left: 0,
        right: 0,
        bottom: 0,
        backgroundColor: 'white',
        paddingBottom: 20,
    },
});
