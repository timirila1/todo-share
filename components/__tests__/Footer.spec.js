import 'react-native';
import React from 'react';
import { Text } from 'react-native';
import { Footer } from '../Footer';
import renderer from 'react-test-renderer';

it('Render Footer', async () => {
    const tree = renderer.create(
        <Footer>
            <Text>Test</Text>
        </Footer>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
