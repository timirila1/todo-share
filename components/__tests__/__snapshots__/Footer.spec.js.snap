// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "borderTopColor": "rgba(0, 0, 0, 0.1)",
      "borderTopWidth": 1,
      "bottom": 0,
      "flex": 1,
      "height": 84,
      "left": 0,
      "paddingBottom": 20,
      "position": "absolute",
      "right": 0,
    }
  }
>
  <Text>
    Test
  </Text>
</View>
`;
